portal_partner_merge/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# See LICENSE file for full copyright and licensing details.
+{'name': 'Portal Partner Merge',
+ 'version': '11.0.1.0.0',
+ 'category': 'Hidden',
+ 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'website': 'http://www.camptocamp.com/',
```

Comment:
IMHO Please change URL as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10